server/lib/transforms/insert-ad.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+const adMarkup = () => `<div class="ad-container"><amp-ad width="300" height="250" type="doubleclick"></amp-ad></div>`;
+
+module.exports = function insertAd($, options) {
+		const paragraph = $('p').eq(3);
```

Comment:
not sure. guess this just tries to make sure that the ad is added after 3rd paragraph? Guess in Next page markup is a lot more complicated thus a more complicated conditional statement?

@quarterto might have some input on this?